src/main/webAuthnService.ts

```diff
@@ -1,9 +1,22 @@
 import { Buffer } from 'buffer/'
 
+import { AuthOptions } from './authOptions'
+import { SignupProfileData } from './models'
 import { encodeToBase64 } from '../utils/base64'
 
 export const publicKeyCredentialType = 'public-key'
 
+export type EmailLoginWithWebAuthnParams = { email: string, auth?: AuthOptions }
+export type PhoneNumberLoginWithWebAuthnParams = { phoneNumber: string, auth?: AuthOptions  }
+export type LoginWithWebAuthnParams =  EmailLoginWithWebAuthnParams | PhoneNumberLoginWithWebAuthnParams
+
+export type SignupWithWebAuthnParams = {
```

Comment:
Naming nitpick: can we just say "SignupWithWebAuth" _or_ "WebAuthnSignup" ? I personally find the "params" part unnecessary 😛

Reply:
mmhh we already use the `Params` suffixes for all the types (eg: `UpdateEmailParams`, `EmailLoginWithPasswordParams`, `LoginWithCredentialsParams` ...)